### DISCUSSION TITLE
Is chat or notification suitable for SWR?

### DISCUSSION THREAD

**Discussion by KeisukeNagakawa** [vercel/swr#2227] [Q&A] [answered]
Hi, I'm thinking of near-realtime system like chat app or notifications button on Nextjs. 

I am saying "near-realtime" in that I'm okay with several minutes of time lag. 

I'm this situation, is is a possible idea for using SWR? I'm new to SWR so I would like some broad opinions. Thanks!

**Comment by dev-afzalansari** [comment, marked as answer]
@KeisukeNagakawa

> I am saying "near-realtime" in that I'm okay with several minutes of time lag.

Yes it is a possible idea you can do this by setting [revalidations options](https://swr.vercel.app/docs/revalidation) according to your need and [mutating](https://swr.vercel.app/docs/mutation) the data locally.

But if doesn't fulfill your need you can go for firebase's realtime databases.
